README.md

```diff
@@ -5,9 +5,18 @@ Personal project of an audio manager & player written in Rust
 
 # Installation
 Environment requirements :
-
  - Rustup - Toolchain management : https://www.rust-lang.org/tools/install
 
+On Ubuntu 22.04 :
+
+install pkg-config and libfontconfig-dev
+
+`sudo apt-get install -y pkg-config libfontconfig-dev`
+
+create a missing file named `audio-player/src/metadata_wrapper/mod.rs` (to avoid the compilation to fail)
+
+`touch audio-player/src/metadata_wrapper/mod.rs` 
+
```

Comment:
I've fixed it with https://github.com/jgarnierGit/mp3-player/commit/21ce079cddd05542e9b19f5b3cc164828136dda5
:)
```suggestion

```